stdlib/3/typing.pyi

```diff
@@ -302,15 +302,15 @@ class Sequence(_Collection[_T_co], Reversible[_T_co], Generic[_T_co]):
     @abstractmethod
     def __getitem__(self, s: slice) -> Sequence[_T_co]: ...
     # Mixin methods
-    def index(self, x: Any, start: int = ..., end: int = ...) -> int: ...
-    def count(self, x: Any) -> int: ...
+    def index(self, value: Any, start: int = ..., stop: int = ...) -> int: ...
```

Comment:
A lot of these should probably also be positional-only.

Reply:
Ideally, yeah. Note that the actual classes in collections.abc don't use positional-only args though, so that would diverge from CPython